overlays/xtensa_nxp_imx8ulp_adsp/gdb/gdb/xtensa-config.c

```diff
@@ -0,0 +1,421 @@
+            0,0,&xtensa_mask10,0,0,0)
+  XTREG(181,759, 1, 1, 1,0x001b,0x0006,-2, 6,0x1010,b11,
+            0,0,&xtensa_mask11,0,0,0)
+  XTREG(182,760, 1, 1, 1,0x001c,0x0006,-2, 6,0x1010,b12,
+            0,0,&xtensa_mask12,0,0,0)
+  XTREG(183,761, 1, 1, 1,0x001d,0x0006,-2, 6,0x1010,b13,
+            0,0,&xtensa_mask13,0,0,0)
+  XTREG(184,762, 1, 1, 1,0x001e,0x0006,-2, 6,0x1010,b14,
+            0,0,&xtensa_mask14,0,0,0)
+  XTREG(185,763, 1, 1, 1,0x001f,0x0006,-2, 6,0x1010,b15,
+            0,0,&xtensa_mask15,0,0,0)
+  XTREG(186,764, 4, 4, 4,0x2006,0x0006,-2, 6,0x1010,psintlevel,
+            0,0,&xtensa_mask16,0,0,0)
+  XTREG(187,768, 1, 4, 4,0x2007,0x0006,-2, 6,0x1010,psum,
+            0,0,&xtensa_mask17,0,0,0)
+  XTREG(188,772, 1, 4, 4,0x2008,0x0006,-2, 6,0x1010,pswoe,
+            0,0,&xtensa_mask18,0,0,0)
+  XTREG(189,776, 1, 4, 4,0x2009,0x0006,-2, 6,0x1010,psexcm,
+            0,0,&xtensa_mask19,0,0,0)
+  XTREG(190,780, 2, 4, 4,0x200a,0x0006,-2, 6,0x1010,pscallinc,
+            0,0,&xtensa_mask20,0,0,0)
+  XTREG(191,784, 4, 4, 4,0x200b,0x0006,-2, 6,0x1010,psowb,
+            0,0,&xtensa_mask21,0,0,0)
+  XTREG(192,788,40, 8, 4,0x200c,0x0006,-2, 6,0x1010,acc,
+            0,0,&xtensa_mask22,0,0,0)
+  XTREG(193,796, 4, 4, 4,0x2011,0x0006,-2, 6,0x1010,dbnum,
+            0,0,&xtensa_mask23,0,0,0)
+  XTREG(194,800, 1, 4, 4,0x2014,0x0006, 1, 6,0x1010,ae_overflow,
+            0,0,&xtensa_mask24,0,0,0)
+  XTREG(195,804,14, 4, 4,0x2015,0x0006, 1, 6,0x1010,ae_sar,
+            0,0,&xtensa_mask25,0,0,0)
+  XTREG(196,808, 1, 4, 4,0x2016,0x0006, 1, 6,0x1010,ae_cwrap,
+            0,0,&xtensa_mask26,0,0,0)
+  XTREG(197,812, 4, 4, 4,0x2017,0x0006, 1, 6,0x1010,ae_bitptr,
+            0,0,&xtensa_mask27,0,0,0)
+  XTREG(198,816, 4, 4, 4,0x2018,0x0006, 1, 6,0x1010,ae_bitsused,
+            0,0,&xtensa_mask28,0,0,0)
+  XTREG(199,820, 4, 4, 4,0x2019,0x0006, 1, 6,0x1010,ae_tablesize,
+            0,0,&xtensa_mask29,0,0,0)
+  XTREG(200,824, 4, 4, 4,0x201a,0x0006, 1, 6,0x1010,ae_first_ts,
+            0,0,&xtensa_mask30,0,0,0)
+  XTREG(201,828,27, 4, 4,0x201b,0x0006, 1, 6,0x1010,ae_nextoffset,
+            0,0,&xtensa_mask31,0,0,0)
+  XTREG(202,832, 1, 4, 4,0x201c,0x0006, 1, 6,0x1010,ae_searchdone,
+            0,0,&xtensa_mask32,0,0,0)
+  XTREG(203,836, 2, 4, 4,0x201d,0x0006, 1, 6,0x1010,roundmode,
+            0,0,&xtensa_mask33,0,0,0)
+  XTREG(204,840, 1, 4, 4,0x201e,0x0006, 1, 6,0x1010,invalidflag,
+            0,0,&xtensa_mask34,0,0,0)
+  XTREG(205,844, 1, 4, 4,0x201f,0x0006, 1, 6,0x1010,divzeroflag,
+            0,0,&xtensa_mask35,0,0,0)
+  XTREG(206,848, 1, 4, 4,0x2020,0x0006, 1, 6,0x1010,overflowflag,
+            0,0,&xtensa_mask36,0,0,0)
+  XTREG(207,852, 1, 4, 4,0x2021,0x0006, 1, 6,0x1010,underflowflag,
+            0,0,&xtensa_mask37,0,0,0)
+  XTREG(208,856, 1, 4, 4,0x2022,0x0006, 1, 6,0x1010,inexactflag,
+            0,0,&xtensa_mask38,0,0,0)
+  XTREG_END
+};
+
+
+
+#ifdef XTENSA_CONFIG_INSTANTIATE
+XTENSA_CONFIG_INSTANTIATE(rmap,16)
+#endif
+
```

Comment:
You need to add this line:

```
xtensa_gdbarch_tdep xtensa_tdep (rmap);
```

in this file. Or else GDB would not build.

Reply:
Done.